src/pages/index.js

```diff
@@ -2,6 +2,8 @@ import Head from "next/head";
 import Image from "next/image";
 import styles from "../styles/Home.module.css";
 import Member from "./member";
+import connectMongo from "../server/mongodb/connectMongo";
+import Test from "../server/mongodb/models/Test";
 
 export default function Home() {
```

Comment:
it doesn't look like it ever grabs the props that were fetched server-side at all-- would be nice to use them here to validate that its actually running well

Reply:
recovered